CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 
 ## 1.0-32
 
+The latest version of the Vision Teaser App includes improvements in algorithms performance, camera calibration process and lane detection accuracy. Also the new version contains new improved ML-models for CV tasks like detection, segmentation and classification. The full list of changes:
+`MapboxVision` v0.12.0
 - Added japanese traffic signs
+- Improved performance by utilizing ARM Advanced SIMD architecture extension
+- Improved lanes detection
+- Improved camera calibration algorithm
+- Released new models version v4
```

Comment:
Seems user does not know about versions of models - suggest to rephrase it. Maybe
`Utilized new ML models that reduce resource utilization`